robot/Cumulus/resources/NPSP.py

```diff
@@ -853,15 +853,15 @@ def wait_for_batch_to_complete(self, path, *args, **kwargs):
      
     @capture_screenshot_on_error                
     def wait_for_batch_to_process(self, batch,status):
-        """Checks every 30 secs for upto 7mins for batch with given status
+        """Checks every 30 secs for upto 9mins for batch with given status
```

Comment:
any reason to increase batch processing time?

Reply:
I noticed that the batch processing time for RD is taking a little longer depending on the environment on Metaci. So increased it slightly. It doesn't mean it will wait the whole time specified. As soon as the completed status is reached the next step gets executed.